pkg/types/vgpu_config.go

```diff
@@ -46,7 +46,7 @@ func (v VGPUConfig) AssertValid() error {
 				return fmt.Errorf("cannot mix time-sliced and MIG-backed vGPU devices on the same GPU")
 			}
 			migBacked = true
-		} else {
+		} else if vgpuType.G <= 0 {
```

Comment:
This is a bit strange. What was the lint error for this?

Reply:
that this was a case for `else if` and not single else.